recipes/rapidyaml/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Git clone submodules
+git submodule update --init --recursive
```

Comment:
it is vendoring ``c4core``

Reply:
You can submit multiple related packages in one PR. It's typically easier to give feedback on related packages as a group as long as the group is around 5 packages for fewer.